lib/sedna/core/federated_learning/federated_learning.py

```diff
@@ -43,6 +42,7 @@ def __init__(self, estimator, aggregation="FedAvg"):
         super(FederatedLearning, self).__init__(
             estimator=estimator, config=config)
         self.aggregation = ClassFactory.get_cls(ClassType.FL_AGG, aggregation)
+        self.transmitter = ClassFactory.get_cls(ClassType.TRANSMITTER, transmitter)
```

Comment:
`test_transmitter` doesn't seem to be registered in sedna.

Reply:
This function is under development. @jaypume 
In the current stage, we enable Plato in Sedna via directly call its server and client.